scripts/functions/utility_system

```diff
@@ -234,23 +234,35 @@ __list_remote_for_s3()
 __rvm_get_user_shell()
 {
   case "${_system_type}:${_system_name}" in
-    (Linux:*|SunOS:*|BSD:*|*:Cygwin)
+    (Linux:*|SunOS:*|BSD:*|Windows:Cygwin)
```

Comment:
Is `_system_type` accompanying `Cygwin` always `Windows`? It seems this line _could_ stay as-is, since `*:Cygwin` won't match `Windows:Mingw` below. Or if it's always `Windows` (which kinda makes sense), never mind!

Reply:
Always `Windows` AFAIK

https://github.com/rvm/rvm/blob/master/scripts/functions/detect_system#L175